nucleus/constants.py

```diff
@@ -1,4 +1,5 @@
 NUCLEUS_ENDPOINT = "https://api.scale.com/v1/nucleus"
+DEFAULT_TIMEOUT = 60
```

Comment:
Nit: can you put the units in the name for constants where the unit is ambiguous. E.g. `DEFAULT_TIMEOUT_SEC`